src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/FileUploadStart.cshtml

```diff
@@ -32,3 +32,13 @@
 {
 <a id="back-link" class="govuk-back-link" href="@Url.Action("FileUploadInform", new { @Model.ProviderId })">Back</a>
 }
+
+
+<style type="text/css" >
```

Comment:
don't think we should have this style section

Reply:
true